ui/StatusQ/src/StatusQ/Core/Theme/Theme.qml

```diff
@@ -147,12 +147,21 @@ QtObject {
     property int tertiaryTextFontSize: 12
     property int asideTextFontSize: 10
 
-    property int xlPadding: 32
-    property int bigPadding: 24
-    property int padding: 16
-    property int halfPadding: 8
-    property int smallPadding: 10
-    property int radius: 8
+    // Responsive properties used for responsive components (e.g. containers)
+    property int xlPadding: defaultXlPadding
+    property int bigPadding: defaultBigPadding
+    property int padding: defaultPadding
+    property int halfPadding: defaultHalfPadding
+    property int smallPadding: defaultSmallPadding
+    property int radius: defaultRadius
+
+    // Constant properties used for non-responsive components (e.g. buttons)
```

Comment:
Hmm, but e.g. `StatusBaseButton` is still using the above (`halfPadding` etc); is that ok?